### ISSUE TITLE
Error Blocked aria-hidden show up when clicking on Popup Proxy (cover mode)

### ISSUE THREAD

**Issue by mityaua** [quasarframework/quasar#17810]
### What happened?

In the Popup Proxy component, if I open the popup in cover mode, the following error appears in the console. However, it looks to work all as expected.

> Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
> Element with focus: div
> Ancestor with aria-hidden:  <div class=​"q-dialog__backdrop fixed-full" aria-hidden=​"true" tabindex=​"-1" style=​"--q-transition-duration:​ 300ms;​">​</div>​

### What did you expect to happen?

I expect no error message when I call popup.

### Reproduction URL

https://quasar.dev/vue-components/popup-proxy/#breakpoint

### How to reproduce?

1. Go to the Quasar's website [https://quasar.dev/vue-components/popup-proxy/#breakpoint](https://quasar.dev/vue-components/popup-proxy/#breakpoint) on Chrome
2. Scroll down to Breakpoint section
3. Open the browser's Developer tools
4. Click on date picker several times (3 or more)
6. You should be able to observe and error in the developer tools.

![Image](https://github.com/user-attachments/assets/b4f3f8e2-21a0-45dc-9435-2ad43d5940f6)

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

### Quasar info output

```Shell

```

### Relevant log output

```Shell
Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: div
Ancestor with aria-hidden:  <div class=​"q-dialog__backdrop fixed-full" aria-hidden=​"true" tabindex=​"-1" style=​"--q-transition-duration:​ 300ms;​">​</div>​
```

### Additional context

Sometimes it happens in dialog mode (with **breakpoint** prop).

**Comment by nataliedemma** [comment]
We have also identified the reported issue. 

**Reproduction URL:** 
https://codepen.io/Natalie-Demmelmaier/pen/dPyZWbW?editors=1011.

Unfortunately, the problem occurs intermittently. It requires the dialog to be opened and closed multiple times to observe the issue:

![Image](https://github.com/user-attachments/assets/7c4b6fcd-f16b-4d1d-a73b-e42509e346b9)

**Comment by mourad1081** [comment]
I also have this issue.

**Comment by qxygene1** [comment]
had similar issue with dialog 

```
  <q-dialog
    transition-show="jump-up"
    transition-hide="jump-down"
    :transition-duration="100"
    position="bottom"
  >
```

when dialog is opening if i click to backdrop then error shows up 

```
Blocked aria-hidden on an element because its descendant retained focus. The focus must not be hidden from assistive technology users. Avoid using aria-hidden on a focused element or its ancestor. Consider using the inert attribute instead, which will also prevent focus. For more details, see the aria-hidden section of the WAI-ARIA specification at https://w3c.github.io/aria/#aria-hidden.
Element with focus: <div.q-dialog__backdrop fixed-full q-transition--fade-enter-active q-transition--fade-enter-to>
Ancestor with aria-hidden: <div.q-dialog__backdrop fixed-full q-transition--fade-enter-active q-transition--fade-enter-to> <div class="q-dialog__backdrop fixed-full" aria-hidden="true" tabindex="-1" style="--q-transition-duration: 100ms;"></div>
```

i solved it with simple style

```
/* global styles */
.q-dialog__backdrop {
  pointer-events: none; /* during transition clicks won't land on it */
}
/* Re-enable after transition for dismiss-on-click-outside to still work */
.q-dialog__backdrop:not(.q-transition--fade-enter-active) {
  pointer-events: auto;
}
```

Maybe be they will fix it on future releases

**Comment by rstoenescu** [comment]
fixed in v2.20.1
